service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseControllerTest.java

```diff
@@ -224,6 +245,55 @@ private OrganisationPolicy blankPolicyFor(SolicitorRole role) {
             .email("solicitor@legacy.com")
             .build();
 
+        final List<Task> tasks = List.of(Task.builder()
```

Comment:
What would you think of having these two fields be internal variables in the init() method (since they're only used there?

Reply:
done